src/Jackett.Common/Definitions/hush.yml

```diff
@@ -98,12 +101,4 @@
             args: ["(ay|kuu|måned|mese|miesiąc|mês|měsíc|месяц|lună|mes|mois|Monat|месец|maand)", "month"]
           - name: re_replace
             args: ["(ay|kuud|måneder|mesi|miesiące|meses|měsíce|месяца|месяцев|luni|meses|mois|Monaten|месеца|maanden)", "months"]
-      downloadvolumefactor:
```

Comment:
are they really no longer using upload and download volume factors?
is it possible all it needed was for the uploadvolumefactor to end with a "*": "1" like the downloadvolumefactor does?